### PR TITLE
Fix #229 and fotmob tests

### LINKED ISSUE #229: fotmob_get_league_matches responds with error
df <- fotmob_get_league_matches(country = "ENG", league_name = "Championship")

responds with the following error

Error in `.f()`:
! Argument 1 must have names.

> packageVersion("worldfootballR")
[1] ‘0.6.2.1000’

> sessionInfo()
R version 4.2.2 (2022-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19044)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.utf8  LC_CTYPE=English_United Kingdom.utf8    LC_MONETARY=English_United Kingdom.utf8
[4] LC_NUMERIC=C                            LC_TIME=English_United Kingdom.utf8    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] worldfootballR_0.6.2.1000 extrafont_0.18            dplyr_1.0.10             

loaded via a namespace (and not attached):
 [1] pillar_1.8.1     compiler_4.2.2   tools_4.2.2      lubridate_1.9.0  jsonlite_1.8.4   lifecycle_1.0.3  tibble_3.1.8     timechange_0.1.1
 [9] pkgconfig_2.0.3  rlang_1.0.6      DBI_1.1.3        cli_3.4.1        rstudioapi_0.14  curl_4.3.3       Rttf2pt1_1.3.11  xml2_1.3.3      
[17] httr_1.4.4       stringr_1.5.0    janitor_2.1.0    generics_0.1.3   vctrs_0.5.1      hms_1.1.2        tidyselect_1.2.0 glue_1.6.2      
[25] snakecase_0.11.0 R6_2.5.1         fansi_1.0.3      tidyr_1.2.1      purrr_0.3.5      readr_2.1.3      tzdb_0.3.0       extrafontdb_1.0 
[33] magrittr_2.0.3   ellipsis_0.3.2   rvest_1.0.3      assertthat_0.2.1 utf8_1.2.2       stringi_1.7.8

### PR DESCRIPTION
**Pull request by tonyelhabr**
`fotmob_get_league_matches()` was failing due to an extra `purrr::map_dfr` that is no longer needed [#229](https://github.com/JaseZiv/worldfootballR/issues/229)

Also, I updated some of the fotmob tests that were failing due to changes in the column names provided by Fotmob.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# worldfootballR 0.6.2.1000
+# worldfootballR (development version)
```
Comment (tonyelhabr) [inline NEWS.md:1]: this should prevent a merge conflict i think

File: R/fotmob_leagues.R [hunk 2]
```diff
@@ -246,11 +246,10 @@ fotmob_get_league_matches <- function(country, league_name, league_id, cached =
 .fotmob_get_league_matches <- function(league_id, page_url) {
   resp <- .fotmob_get_league_resp(league_id, page_url)
   rounds <- resp$matches$data$matchesCombinedByRound
-  rounds %>%
-    purrr::map_dfr(
```
Comment (tonyelhabr) [inline R/fotmob_leagues.R:250]: there used to be an extra layer of nesting here that is no longer there, so we just need one `map_dfr`

File: NEWS.md (hunk 1 above)
Comment (JaseZiv) [reply NEWS.md:1]: Almost, however there are still some merge conflicts. I’m not in front of my laptop for a while - are you able to fix these?

**Review by JaseZiv — CHANGES_REQUESTED** [review]
We just need to fix the merge conflict

File: NEWS.md (hunk 1 above)
Comment (tonyelhabr) [reply NEWS.md]: alright it should be resolved now

**Review by JaseZiv — APPROVED** [review]

**Pull request merged**